### ISSUE TITLE
sddm not starting on arch

### ISSUE THREAD

**Issue by sashazamechek** [sddm/sddm#1940]
when i start my computer sddm is a black screen with nothing on it but a mouse cursor. 
i have a nvidia graphics card.

**Comment by 69NoraV** [comment]
I'm having the same issue with an AMD GPU and arch. I do not have the issue however on my laptop with an intel GPU and latest packages installed (arch still)

**Comment by Redeeman** [comment]
try latest from git, I think i had same issue, though multi-seat is involved, and worked from git. I have amdgpu aswell

**Comment by m4chyn3** [comment]
in advance, excuse my English.

I am new in this linux world, and yesterday I found myself with this same problem.
I started to change the theme of my Arch Linux, and when I rebooted, the screen was black and only the cursor was visible.
So much looking for solutions and trying different commands, I found myself watching a video of youtuber Zatiel where he showed the sddm.conf file. and with this I managed to give solution to my Arch Linux.
Steps to follow.
When you are in this screen (Black with only the cursor) press. Ctrl+Alt+F2 or Ctrl+Alt+F3. This will open a tty.
Enter Username and Password.
Type: 
sudo nano /etc/sddm.conf
This command will open the sddm.conf file with nano text editor.
If the text is blank (without any content, this would be the problem of not showing the login screen).

Then proceed to type:
[Autologin]
Session=plasma

Then, press
Ctrl+O to save.
Ctrl+X to exit.

Reboot.

And this is how I was able to solve my problem.
I hope to have been of help. 
Thanks and see you next time. I still have a lot to learn about Arch Linux.

**Comment by dinno7** [comment]
[SOLVED]
My configurations was correct, but I do not know what I did that caused it to go wrong and face me this problem. I have Arch Linux, KDE Plasma, Wayland and I use SDDM.
Yesterday, when I booted my system, I could not see the SDDM login page. I had to use `CTRL + ALT + F5` to switch to a TTY, run `systemctl restart sddm.service`, and then access the SDDM login page temporarily(You can perform the below steps without restarting SDDM and access it temporarily. I needed to force access to my system, so did it).

By the way, I resolved my problem with the following steps:

> [!NOTE]
> SDDM uses `/etc/sddm.conf.d/kde_settings.conf` by default in my case, and my `/etc/sddm.conf` was empty.
> **Get a backup of your related settings before changing anything.**

```bash
sudo sddm --example-config > /etc/sddm.conf
```

Then I opened `/etc/sddm.conf` and made the following changes:

```bash
sudo vim /etc/sddm.conf
```

```
[Autologin]
Session=plasma

[General]
DisplayServer=wayland
```

After saving the changes, I rebooted the system and Done.

**Comment by Vogtinator** [comment]
> sudo sddm --example-config > /etc/sddm.conf

That is incorrect use of `sudo` and will override all defaults from your distro.

I assume that's one of the reasons why it appears to work. You'd have to figure out which of the changes triggered it.

**Comment by dinno7** [comment]
I know that using `sudo sddm --example-config > /etc/sddm.conf` override the defaults from distro,  
but as I mentioned, `/etc/sddm.conf` was empty before running this command. It was using `/etc/sddm.conf.d/kde_settings.conf`.

**Comment by Vogtinator** [comment]
Distro defaults are in `/usr/lib/sddm/sddm.conf.d/`

**Comment by dinno7** [comment]
Ah, I didn’t know that. Thanks for letting me know.  
Whenever I figure out which changes caused the issue, I’ll add it here.
